README.md

```diff
@@ -0,0 +1,67 @@
+# Sloboda EDU
+
+## About
+
+This repo represents [the front-page of Sloboda EDU project](https://sloboda-edu-dashboard.herokuapp.com)
+
+## Getting started
+
+1. Install ruby & choose the project's ruby version (you can simply use [RVM](https://rvm.io/) to do that):
+
+  ```bash
+    rvm install ruby-2.2.3
+  ```
+  ```bash
+    use ruby-2.2.3
+  ```
+  
+2. Create gemset & switch to it (if you use [RVM](https://rvm.io/)):
+  
+  ```bash
+    rvm gemset sloboda-edu-dashboard
+  ```
+  ```bash
+    rvm use sloboda-edu-dashboard
+  ```
+  
+3. Install [Bundler](http://bundler.io/):
+
+  ```bash
+    gem install bundler
+  ```
+  
+4. Install gems:
+
+  ```bash
+    bundle install
+  ```
+
+5. Rename existing `database.yml.sample` to `database.yml` (it stores the db configs);
```

Comment:
It's better to copy rather than rename, to not affect the git:
`cp config/database.yml.sample config/database.yml`